src/main/java/com/kruize/service/HealthService.java

```diff
@@ -0,0 +1,53 @@
+/*******************************************************************************
+ * Copyright (c) 2019, 2020 IBM Corporation and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+package com.kruize.service;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class HealthService extends HttpServlet
+{
+    public static final int STATUS_UP = 1;
+    public static final int STATUS_DOWN = 0;
+
+    private static int CURRENT_STATUS = STATUS_DOWN;
+
+    public static int getCurrentStatus()
+    {
+        return CURRENT_STATUS;
+    }
+
+    public static void setCurrentStatus(int currentStatus)
+    {
+        CURRENT_STATUS = currentStatus;
```

Comment:
Can we have a check on the setter function please. In general all setters should validate what is being set.